src/js/utils/ValidatorUtil.js

```diff
@@ -34,22 +34,22 @@ var ValidatorUtil = {
   },
 
   isInteger: function (value) {
-    const number = parseFloat(value);
-
-    return Number.isInteger(number);
+    return ValidatorUtil.isNumber(value) &&
+      Number.isInteger(parseFloat(value));
```

Comment:
+1


> On Aug 12, 2016, at 1:29 PM, Orlando Hohmeier <notifications@github.com> wrote:
> 
> In src/js/utils/ValidatorUtil.js:
> 
> > @@ -34,22 +34,22 @@ var ValidatorUtil = {
> >    },
> >  
> >    isInteger: function (value) {
> > -    const number = parseFloat(value);
> > -
> > -    return Number.isInteger(number);
> > +    return ValidatorUtil.isNumber(value) &&
> > +      Number.isInteger(parseFloat(value));
> We probably should rename these methods as they intentional use parseFloat to also handle number like strings. How do you feel about isNumericValue, isIntegerValue , and isNumericValueInRange?
> 
> —
> You are receiving this because you commented.
> Reply to this email directly, view it on GitHub, or mute the thread.
>